javascript/javascript3/week3/shopping.js

```diff
@@ -0,0 +1,93 @@
+class Product {
+    constructor(name, price) {
+        this.name = name;
+        this.price = price;
+    }
+    convertToCurrency(currency) {
+        if (currency.toLowerCase().includes('dollar')) {
+            return this.price * 0, 16
+        } else if (currency.toLowerCase().includes('euro')) {
+            return this.price * 0, 13
+        } else if (currency.toLowerCase().includes('pound')) {
+            return this.price * 0, 12
+        } else {
+            return this.price
+        }
+    }
```

Comment:
First of all, this function is not used anywhere, so it should either be deleted or used.
second, it functions wrongly. You times the prices with 0, thus it will always return 12,13 or 16 which is not quite converting currencey.

Reply:
thanks.. i mentioned ',' instead of '.' ... i fixed now .. thanks